docs/ops/state/savepoints.md

```diff
@@ -120,6 +120,10 @@ This will atomically trigger a savepoint for the job with ID `:jobid` and cancel
 
 If you don't specify a target directory, you need to have [configured a default directory](#configuration). Otherwise, cancelling the job with a savepoint will fail.
 
+<div class="alert alert-warning">
```

Comment:
Should be something like:
```
<div class="alert alert-warning">
    <strong>Attention:</strong> <code>targetDirectory</code> has to be a location accessible by both the JobManager(s) and TaskManager(s), e.g., a location on a distributed file system.
</div>
```
because markdown is not rendered within HTML tags.

Also, the *warning* you have added is currently in the *Cancel Job with Savepoint* section. I think it should be moved to the section above (*Trigger a Savepoint*), or somewhere else since it does not only apply to the cancelation case.

@uce

Reply:
👍